src/cli/onefuzz/debug.py

```diff
@@ -613,6 +615,85 @@ def _query_libfuzzer_execs_sec(
 
         return self.onefuzz.debug.logs._query_parts(query_parts, timespan=timespan)
 
+    def get(
+        self,
+        job_id: str,
+        task_id: Optional[str],
```

Comment:
we should support getting the logs by only specifying the task_id that is the most likely use case